algorithmic_efficiency/workloads/imagenet/imagenet_jax/input_pipeline.py

```diff
@@ -209,22 +213,19 @@ def create_split(split,
                  aspect_ratio_range=(0.75, 4.0 / 3.0),
                  area_range=(0.08, 1.0)):
   """Creates a split from the ImageNet dataset using TensorFlow Datasets."""
+  del num_batches
   if split == 'eval_train':
-    split = 'train'
+    split = 'train[:50000]'
```

Comment:
good catch. should we instead in the caller function, pass in split='train[:{num_eval_train_examples}]'?

Reply:
Makes a lot of sense, already changed it.